src/deployment/misc.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * (c) 2021, Micro:bit Educational Foundation and contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+// Miscellaneous items that haven't been properly integrated into the branding.
+export const backgroundColorTerm = "#333333";
+export const defaultCodeFontSizePt = 14;
```

Comment:
Some of these feel likely to be unused here.

Reply:
I have removed misc.ts here -> https://github.com/microbit-foundation/ml-trainer/pull/267/commits/39e456b54400e5b1441304b1eba5180bc5839155